qa/L0_lifecycle/lifecycle_test.py

```diff
@@ -284,9 +285,15 @@ def test_parse_error_no_model_config(self):
                     "' to be ignored due to polling failure")
 
             except Exception as ex:
-                self.assertTrue(ex.message().startswith(
-                    "Request for unknown model: 'graphdef_float32_float32_float32' is not found"
-                ))
+                self.assertIn(
+                    "Request for unknown model: 'graphdef_float32_float32_float32' is not found",
+                    ex.message())
+                with open(os.environ["SERVER_LOG"]) as log:
```

Comment:
If we want to check if certain message is in the server log, the convention is to do so in the bash script, look up "grep" for usage.

Reply:
Thanks for this tip. I'll update the ticket to reflect the convention. If we ever want to think about changing the convention in the future, it'd likely be helpful to provide the exact error via the Python script (logs) rather than "Test Failed," since there are multiple points of failure.

Reply:
That actually reminds me something I found when looking at the other lifecycle ticket. The client log is actually not being printed (cat) at the end in the case of error (while other tests do), you should probably fix that as you are working on it. Note that you may need to remove the `rm -f $CLIENT_LOG` so the client log at the end is complete, or you can clear the log before every test and print it as soon as the test failed. Either way is fine as long as we do report the logs on failure

Reply:
Added.